builder/actions/install.py

```diff
@@ -99,16 +99,19 @@ def export_compiler(compiler, env):
         return
 
     toolchain = env.toolchain
-    cc_path = toolchain.compiler_path()
-    cxx_path = toolchain.cxx_compiler_path()
-    if cc_path:
-        env.shell.setenv('CC', cc_path)
-    else:
-        print('WARNING: C compiler {} could not be found for export'.format(compiler))
-    if cxx_path:
-        env.shell.setenv('CXX', cc_path)
-    else:
-        print('WARNING: CXX compiler {} could not be found for export'.format(compiler))
+    if not env.shell.getenv('CC'):
```

Comment:
We should never override the environment if it's specified. Or should we check to see if `--compiler` was specified on the command line and override it then?